scripts/exrrfs_analysis_enkf.sh

```diff
@@ -402,6 +410,13 @@ countdiag=$(ls diag*conv* | wc -l)
 if [ $countdiag -gt $nens ]; then
   ${APRUN} ${EXECrrfs}/$pgm < enkf.nml >>$pgmout 2>errfile
   export err=$?; err_chk
+
+# DELETING ${enkfanal_nwges_dir} - should ${pgmout} be saved elsewhere?
+#  cp ${pgmout} ${enkfanal_nwges_dir}/.
+#  if [ ! -d ${GESROOT}/enkf_diag ]; then
+#    mkdir -p ${GESROOT}/enkf_diag
+#  fi
+#  cp ${pgmout} ${GESROOT}/enkf_diag/${stdout_name}.$vlddate
 else
```

Comment:
Same comment here, these lines can be removed.

Reply:
Yes. They can be removed.